### ISSUE TITLE
Race Formatting

### ISSUE THREAD

**Issue by ExplosiveRunes** [SqueeG/awesomeTome#28]
I've finally got the skeleton of a race format that I think looks presentable. I used the aasimar as an example. Its currently using badawful description text that I wrote. If the format seems okay, I'll standardize and improve it, and provide an examplerace.tex.

**Comment by Tarkisflux** [comment]
I like the formatting quite a bit actually. Since you're already familiar with it, here's what I'd want to see tweaked:
- More vertical divider space between the intro text and the table.
- In my pdf I see a bit of trailing line space after the bolded parts of the fluff bits. Not sure if we can tighten that up at all, but we should try (unless I'm seeing things).
- The table should probably start on the first line after the blank space following the quote, not on the same line as the race name.
- The dark grey subsection headers seem weird. They're not bolded and they're right in the middle of everything else, so their import is lost a bit. Maybe a bit of blank space between them and the precedding table subsection, and also bolding the text there.
- Speed entry is missing a colon.
- Table should standardize vision entries, so there is one for every race written. It may be redundant to write "Vision: Normal" a bunch of times, but I'd like to get the tables standardized if we're going to have them.
- Not sure if the special abilities should be different rows in the table or just part of the same row for color consistency. They already have the dots to help keep them separated after all. Might be worth a look.
- Not sure if the location of the age and height/weight sections should be fixed. For some races, it might make sense to put it on the right, for others with really long abilities the left might be a better use of space. Is that section largely droppable if we wanted to move it on a case by case basis?

**Comment by ExplosiveRunes** [comment]
I'm still learning how to use minipages, which is what I used to make this, but here's what I think I could do to address your suggestions.
- Place the race header and quote in a minipage that spans the length of the page, above the two below it. this way both the description text and the table would appear below them and be even.
- I think the text spacing gets weird in the minipage because of how the \ability command interacts with it. I should be able to get rid of that extra space.
- **Weird Grey Headers:** I forgot to bold them, but thats only part of the problem. I think it (and all the tables) would look better if we actually picked a set of three colors to use in them. The greys were always placeholders anyway.
- **Missing Colon:** Derp. Well, once I hammer it out we should be able to set the race formatting up to be like the class formatting, in that it will mostly build itself. Should mitigate typos.
- **Vision Entries:** No, I think it's a good idea to explicitly call out what forms of sense all the races have (Though I'd have it be "Standard" vision"). I'd also suggest that we give better rules than standard 3.5 for those things when we get to the "Exploring and Adventuring" chapter (or whatever we call it), and place it alongside the light sources and whatnot.
- **Seperate Blocks for Special Abilities:** Yeah, I could see a race that has lots of minor special abilities (I'm looking at you, dwarf) ending up looking really fucking busy if it keeps alternating colors. We can make it a single block.
- **Age and Whatnot:** Those area already separate tabulars, and would fit equally well in either minipage (or, for that matter, in a standard twocolumn environment). They can be moved by simply cut and pasting them.

I'll probably start working on this during the coming weekend, so feel free the keep spitballing stuff at me.

**Comment by Tarkisflux** [comment]
Nah, that sounds good (even 'standard vision' and additional stuff elsewhere). I really liked the formatting and think it does a great job of highlighting the mechanical bits for people who don't want to read the rest of it (which I support), so it was all nitpick stuff. 

Though if the age and height/weight are already separate, can we add a few points of blank line between them and the other tables?

**Comment by SqueeG** [comment]
Bleh, what methods do you two use to preview specific pages with out compiling the whole damn project?

**Comment by Tarkisflux** [comment]
I have a separate working folder where I took the whole project and
stripped out all the content that I don't want rendered. Then I just
compile that much reduced thing.  Soon as I get around to striping out the
formatting and putting it in chapter specific sub pages it should be even
easier to do smaller bits if you wanted by just including the formatting
page elsewhere.

**Comment by ExplosiveRunes** [comment]
I do nearly the same thing, I just comment out the stuff I don't want rendered until I'm finished working.

**Comment by SqueeG** [comment]
That table is wicked. Love it.

**Comment by ExplosiveRunes** [comment]
As of the most recent commint ("sometime, git, I hate you") there's all sorts of stuff like 
- >>>>>>> 513934aaa27cc71439ca0cda1d5a44c3ff20f12a
- =======
- <<<<<<< HEAD

What is this, was it intentional?

**Comment by Tarkisflux** [comment]
Which file is that in?

**Comment by ExplosiveRunes** [comment]
weapons.tex and tome-ref-doc.tex

**Comment by SqueeG** [comment]
I, apparently, did not satisfy the merge conflict appropriately.

"These are conflict markers - git uses them to indicate where your local changes conflicted with those pulled from the repository. See how to merge git conflicts"

http://stackoverflow.com/questions/6980873/git-inserts-junk-meta-tags-inside-the-files-after-pull

**Comment by Tarkisflux** [comment]
Yeah, I think it was my fault. It's been sorted already.

New race comments:
- Indents yes (because we were using them for the doc I thought), extra
  line breaks no.
- Still want to see the racial special things bullet points as part of the
  same color section, with a bolded header.
- I think we should remove the "lb." from the weight modifier cells. It
  better matches with the height mod lack of unit (and also avoids (lb)^2 as
  a unit if read literally).

I'll take a look at the file in a couple of days anyway (to play with table
formatting and see how that minipage thing works), so I may just do those
while I'm in there.
